### PR TITLE
OS-8534 SmartOS UI minimum viable product

### PR DESCRIPTION
**Pull request by bahamat**

### CODE REVIEW

**Comment by robbyandrews** [comment]
In the first and third paragraphs sometimes UI is capitalized and sometimes lowercase, so minor suggestion of just uniformly capitalizing UI in those parts.

File: docs/web-interface.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# SmartOS Web Interface
+
+SmartOS now has a web based user interface available. This UI is most directly
+simply exposing SmartOS primitives via a web interface for the purposes of
+managing instances and aims to make simpler tasks much more approachable for new
+users, as well as make more complex tasks easier for advanced users.
+
+In all cases, the web ui simply exposes what's already available in SmartOS
```
Comment (robbyandrews) [inline docs/web-interface.md:8]: ui to UI suggestion

File: docs/web-interface.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+in a more inviting way.
+
+Note that while we have tried to make sure SmartOS features are represented in
+the web ui, not all features are yet exposed. And due to the nature of SmartOS
```
Comment (robbyandrews) [inline docs/web-interface.md:12]: ui to UI suggestion here as well.

**Review by robbyandrews — APPROVED** [review]

File: docs/web-interface.md (hunk 1 above)
Comment (danmcd) [reply docs/web-interface.md:8]: I concur.

**Review by danmcd — COMMENTED** [review]
Fix Robby's comments and I'm good.

Also, not a blocker now, but maybe in the future include some pictures?

**Review by danmcd — APPROVED** [review]

**Pull request merged**